iothub/device/src/IotHubDeviceClient.cs

```diff
@@ -183,7 +183,7 @@ protected override void Dispose(bool disposing)
             base.Dispose(disposing);
         }
 
-        internal override void AddToPipelineContext()
+        private protected override void AddToPipelineContext()
```

Comment:
private protected is another special combo, like protected internal. It means only to types in this library that inherit from me.